src/txn/txn.c

```diff
@@ -441,20 +441,61 @@ __wt_txn_config(WT_SESSION_IMPL *session, const char *cfg[])
 	if (cval.len > 0) {
 #ifdef HAVE_TIMESTAMPS
 		wt_timestamp_t ts;
+		WT_TXN_GLOBAL *txn_global;
+		char timestamp_buf[2 * WT_TIMESTAMP_SIZE + 1];
+		bool round_to_oldest;
 
+		txn_global = &S2C(session)->txn_global;
 		WT_RET(__wt_txn_parse_timestamp(session, "read", &ts, &cval));
-		WT_RET(__wt_timestamp_validate(session,
-		    &ts, &cval, true, false, false));
-		__wt_timestamp_set(&txn->read_timestamp, &ts);
+		WT_RET(__wt_config_gets_def(session, cfg, "round_to_oldest", 0,
```

Comment:
The convention followed is that if you need to wrap to multiple lines, the later lines get more arguments. So in this case : 
```
WT_RET(__wt_config_gets_def(session,
    cfg, "round_to_oldest", 0, &cval));
```